src/version.h

```diff
@@ -1,3 +1,4 @@
 #define KEYDB_REAL_VERSION "0.0.0"
```

Comment:
Some redis modules check this to verify you're not trying to run them on an outdated version of redis (or KeyDB). This should be updated because if this is set to 0.0.0, those modules will fail to load.

Reply:
@JohnSully You mentioned that this should be 0.0.0 in unstable, right? How does that resolve with what Vivek is saying?

Reply:
Redis sets their unstable versions to 255.255.255 to alleviate the problem, maybe we could do the same in KeyDB?

Reply:
This is a recent change for Redis unstable, lets follow them and continue with 255.255.255 (and update the hex version accordingly).